packages/shared/src/utils/extractMirrorXyzUrl.ts

```diff
@@ -1,5 +1,10 @@
 export const extractMirrorXyzUrl = (tokenMetadata: string) => {
-  const metadataObj = JSON.parse(tokenMetadata);
+  let metadataObj;
+  try {
+    metadataObj = JSON.parse(tokenMetadata);
+  } catch (e) {
+    return '';
+  }
```

Comment:
fixes a random error on the dev feed where metadata for a token doesn't exist

<img width="531" alt="Screenshot 2023-11-04 at 1 16 19 AM" src="https://github.com/gallery-so/gallery/assets/12162433/f989fa7c-2268-4ffe-96d8-583f817e20be">